package.json

```diff
@@ -2,6 +2,10 @@
   "name": "threshold-subgraph",
   "license": "GPL-3.0-or-later",
   "scripts": {
+    "lint": "eslint . --ignore-path .gitignore",
```

Comment:
I wonder if eslint and prettier ignore the `yarn.lock` file by default 🤔. My point is `.gitignore` doesn't include `yarn.lock` file.

Reply:
Good point! I have added a file with patterns to ignore files for ESLint and Prettier. Check https://github.com/threshold-network/subgraph/pull/14/commits/d8311473dac3238feff81fe81c37dc7e1a6a4cdf